CONTRIBUTING.md

```
@@ -113,6 +113,16 @@ pnpm playground:react
 pnpm playground:javascript
 ```
 
+`Standalone no-bundler reproduction`:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)